circle.yml

```diff
@@ -1,5 +1,6 @@
 machine:
   pre:
+    - brew update
```

Comment:
Please explain in the commit message why this is necessary, because AFAIK `brew install` does an auto-update.

Reply:
Some useful threads which I found were -> https://discuss.circleci.com/t/homebrew-must-be-run-under-ruby-2-3-runtimeerror/17232 and https://github.com/Homebrew/brew/issues/3299#issuecomment-335837083